spec/features/user_can_add_new_horse_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../spec_helper'
+
+describe "When user fills out form and clicks submit" do
+  it "the database adds a new horse" do
+    Breed.create(name: "Palimino")
+    Jockey.create(name: "Adolf")
+
+    visit('/horses/new')
+
+    fill_in('horse[name]', :with=>"Dufus")
+    fill_in('horse[total_winnings]', :with =>3)
+    select('Adolf', from: "horse[jockey_id]")
+    select('Palimino', from: "horse[breed_id]")
+
+    click_button("Create New Horse")
+    horse = Horse.find(1)
+
+    expect(horse).to be_valid
```

Comment:
These kind of assertions belong with a model test. For a feature test you're trying to assert what the page looks like, not only what the model looks like. 

You're expectations will look more like this:

```ruby
within(".horse_info") do
  expect(page).to have_content("Dufus")
  expect(page).to have_content("total winnings: 3")
  expect(page).to have_content("Adolf")
  expect(page).to have_content("Palamino")
end
```